File: JS/largestPalindromeProduct.js
function largestPalindrome(){

    var arr = [];    
    for(var i =999; i>100; i--){
        for(var j = 999; j>100; j--){
            var mul = j*i;
            if(isPalindrome(mul)){
                arr.push(j * i);
            }
        }
    }

    return Math.max.apply(Math, arr);
}

function isPalindrome(i){
    return i.toString() == i.toString().split("").reverse().join("");
}

console.log(largestPalindrome());

File: Dart/04. Largest palindrome product/Largest palindrome product.dart
//Euler 04


bool isPalindrome(String str) {
  String s = str.toLowerCase().replaceAll(RegExp(r'[\W_]'), '');
  return s == s.split('').reversed.join('');
}

void main() {
  int large = 0;
  var x;
  for (var i = 100; i < 1000; i++) {
    for (var j = i; j< 1000; j++){
      x = (i*j).toString();
              if (isPalindrome(x)){
                if(i*j > large){
                  large = i*j;
                }
                
              }

    }
    
  }
  print(large);
}